servlet/6.1/_index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Jakarta Servlet 6.1 (under development)"
+date: 2023-05-23
+summary: "Release for Jakarta EE 11"
+---
+Jakarta Servlet defines a server-side API for handling HTTP requests and responses.
+
+The release removes references to the SecurityManager and provides various small enhancements and clarifications.
+
+### New features, enhancements or additions
+* Allow control of status code and response body when sending a redirect
+* Add a query string attribute to error dispatches
+* Add constants for new HTTP status codes
+* Add overloaded methods that use `CharSet` rather than `String`
+* Add `ByteBuffer` support to `ServletInputStream` and `ServletOutputStream`
+* Various clarifications
+
+### Removals, deprecations or backwards incompatible changes
+* All references to the SecurityManager and associated APIs have been removed
+
+### Minimum Java SE Version
+**Java SE 11 or higher**
+
+# Details
+
+* [Jakarta Servlet 6.1 Release Record](https://projects.eclipse.org/projects/ee4j.servlet/releases/6.1)
+
```

Comment:
This URL does not exist.

Reply:
It is redirected to
https://projects.eclipse.org/projects/ee4j.servlet/governance